packages/cf-component-checkbox/src/Checkbox.js

```diff
@@ -1,10 +1,27 @@
+// @flow
+
 const React = require('react');
 const {PropTypes} = React;
 
+type InputElementEvent = Event & {
+  target: HTMLInputElement
```

Comment:
Should replace all of these with `currentTarget` since this is unsound due to bubbling.